src/registrar/management/commands/create_federal_portfolio.py

```diff
@@ -146,10 +171,15 @@ def create_suborganizations(self, portfolio: Portfolio, federal_agency: FederalA
             TerminalHelper.colorful_logger(logger.warning, TerminalColors.FAIL, message)
             return
 
-        # Check if we need to update any existing suborgs first. This step is optional.
+        # Check for existing suborgs on the current portfolio
         existing_suborgs = Suborganization.objects.filter(name__in=org_names)
         if existing_suborgs.exists():
-            self._update_existing_suborganizations(portfolio, existing_suborgs)
+            # QUESTION FOR REVIEWERS: Should we be doing this too?
+            # existing_suborgs.filter(portfolio__isnull=True).update(
+            #     portfolio=portfolio
+            # )
+            message = f"Some suborganizations already exist for portfolio '{portfolio}'."
```

Comment:
Hmm..  I didn't know that a suborg could exist without a portfolio. This questions seems discretionary since it is not part of the acs.